test/__init__.py

```diff
@@ -0,0 +1,6 @@
+from unittest import main
+
+# from katello.content_view_test import ContentViewTest
```

Comment:
This is a placeholder, but it acts as an example of how to add test classes. The methods in the test classes must start with `test_`, one of the python unittest conventions.

Reply:
```
$ make test
python -m unittest test

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```
Impressive ;)
Well could you at least add one test to prove the concept? (And as a template to copy an paste...)
Also, i want to see VCRpy in action. 🎥

Reply:
Yes, this PR is WIP (should have set the title).